src/python_spy.rs

```diff
@@ -128,56 +128,56 @@ impl PythonSpy {
     }
 
     /// Gets a StackTrace for each thread in the current process
-    pub fn get_stack_traces(&mut self) -> Result<Vec<StackTrace>, Error> {
+    pub fn get_stack_traces(&mut self, gil_only: bool) -> Result<Vec<StackTrace>, Error> {
```

Comment:
Rather than pass the `gil_only` flag as a parameter here - can we do the same as other config options and just read from the `self.config.gil_only` member ? 


```suggestion
    pub fn get_stack_traces(&mut self) -> Result<Vec<StackTrace>, Error> {
```

Reply:
Done. It's a much smaller diff now :)